reference/6/Microsoft.PowerShell.Core/Disable-PSRemoting.md

```diff
@@ -20,220 +20,250 @@ Disable-PSRemoting [-Force] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Disable-PSRemoting** cmdlet prevents users on other computers from running commands on the local computer.
 
-**Disable-PSRemoting** blocks remote access to all session configurations on the local computer.
-This prevents remote users from creating temporary or persistent sessions to the local computer.
-**Disable-PSRemoting** does not prevent users of the local computer from creating sessions (**PSSessions**) on the local computer or remote computers.
-
-To re-enable remote access to all session configurations, use the Enable-PSRemoting cmdlet.
-To enable remote access to selected session configurations, use the *AccessMode* parameter of the Set-PSSessionConfiguration cmdlet.
-You can also use the Enable-PSSessionConfiguration and Disable-PSSessionConfiguration cmdlets to enable and disable session configurations for all users.
+The `Disable-PSRemoting` cmdlet prevents users on other computers from running PSCore commands on
+the local computer.
+
+`Disable-PSRemoting` blocks remote access to all PSCore session configurations on the local
+computer. This prevents remote users from creating temporary or persistent PSCore sessions to the
+local computer. `Disable-PSRemoting` does not prevent remote users from creating Windows PowerShell
+sessions to the local computer. To disable all PowerShell session configurations, including Windows
+PowerShell configurations, run `Disable-PSRemoting` in Windows PowerShell on the local computer.
+`Disable-PSRemoting` does not prevent users of the local computer from creating sessions
+(**PSSessions**) on the local computer or remote computers.
+
+To re-enable remote access to all PSCore session configurations, use the `Enable-PSRemoting` cmdlet.
+To re-enable remote access to all session configurations, including Windows PowerShell
+configurations, run `Enable-PSRemoting` in Windows PowerShell on the local computer. To enable
+remote access to selected session configurations, use the *AccessMode* parameter of the
+`Set-PSSessionConfiguration` cmdlet. You can also use the `Enable-PSSessionConfiguration` and
+`Disable-PSSessionConfiguration` cmdlets to enable and disable session configurations for all users.
 For more information about session configurations, see [about_Session_Configurations](About/about_Session_Configurations.md).
 
-In Windows PowerShell 2.0, **Disable-PSRemoting** prevents all users from creating user-managed sessions to the local computer.
-In Windows PowerShell 3.0, **Disable-PSRemoting** prevents users on other computers from creating user-managed sessions on the local computer, but allows users of the local computer to create user-managed loopback sessions.
+In Windows PowerShell 2.0, `Disable-PSRemoting` prevents all users from creating user-managed
+sessions to the local computer. In Windows PowerShell 3.0 - 5.1, and PSCore, `Disable-PSRemoting`
```

Comment:
The command is the same for all versions. No need to call out the versions here.